tests/webview/ui/test_content.py

```diff
@@ -766,3 +765,38 @@ def test_chapter_review_version_matches_book_version(webview_base_url, selenium,
 
     # THEN the chapter review version matches the book version
     assert content.page_version == content.book_version
+
+
+@markers.webview
+@markers.test_case('C195064')
+@markers.nondestructive
+@markers.parametrize('ch_review_id', ['4fGVMb7P@1'])
+def test_go_to_first_page(webview_base_url, selenium, ch_review_id):
+    # GIVEN the webview base url, a chapter review id, and the Selenium driver
+    content = Content(selenium, webview_base_url, id=ch_review_id).open()
+    book_content = content.book_contents.wait_for_region_to_display()
+    books = book_content.books
+
+    # WHEN we click the link to the first book
+    book = books[0].go_to_book()
+
+    # THEN the chapter should be the very first module 1.1
+    assert type(book) == Content
+    assert book.chapter_section == '1.1'
+
+    assert book.title == 'English Home Language Grade 5'
```

Comment:
Try not to hardcode things into tests or in any code if you can avoid it. I believe the title of this book is exactly what the text of first link is. Use that text to do the assert later. Do not hardcode things plz.